### ISSUE TITLE
Temporarily require libmicrohttpd?

### ISSUE THREAD

**Issue by simondlr** [ethereum/homebrew-ethereum#11]
Hi.

I ran:
brew tap ethereum/ethereum
brew install ethereum --build-from-source

and it exited, not finding "libmicrohttpd".

However I see it is jsonrpc4 (which it seems it will be upgraded to).

So, temporarily include libmicrohttpd as dependency in ethereum.rb until jsonrpc4 is implemented?

**Comment by caktux** [comment]
What exited? libmicrohttpd will only be needed with jsonrpc4 which is currently not being used

**Comment by simondlr** [comment]
I tried recreating the issue, but alas I can't seem to. It doesn't quite make sense. Several times I ran "brew install ethereum --build-from-source" for it to fail, stating it requires libmicrohttpd as dependency. Once I installed it manually ("brew install libmicrohttpd"), did it only finish properly.

I uninstalled libmicrohttpd just now and it worked (reinstall ethereum --build-from-source). I also uninstalled both jsonrpc & jsoncpp and it ran fine again.

Clearly libmicrohttpd is only required in jsonrpc4, and that's commented out, so I have no idea how it slipped in? Or why the dependency error was actually being thrown.

I'll close the issue for now and re-open it if I experience it again.
